src/main/scala/com/kinja/soy/SoyWrites.scala

```diff
@@ -75,6 +75,13 @@ trait DefaultSoyWrites {
   }
 
   /**
+   * Converter for Float types.
+   */
+  implicit object FloatSoy extends SoyWrites[Float] {
```

Comment:
You mean [here](https://github.com/gawkermedia/kinja-mantle/blob/master/subprojects/macros/src/main/scala/com/kinja/mantle/model/Feature.scala#L65)?